packages/data-driver-postgres/src/query/limit.ts

```diff
@@ -0,0 +1,15 @@
+import type { SqlStatement } from '@directus/data-sql';
+
+/**
+ * Generate the `LIMIT x` part of a SQL statement.
+ *
+ * @param query - The abstract query
```

Comment:
For consistency (we could also have the hyphen everywhere)
```suggestion
 * @param query The abstract query
```